crates/red_knot_test/src/assertion.rs

```diff
@@ -67,16 +67,16 @@ impl InlineFileAssertions {
         }
     }
 
-    fn locator(&self) -> Locator {
-        Locator::with_index(&self.source, self.lines.clone())
+    fn range_text(&self, range: impl Ranged) -> &str {
+        &self.source[range.range()]
```

Comment:
`range.range()` reads a little oddly -- maybe this?

```suggestion
    fn range_text(&self, ranged: impl Ranged) -> &str {
        &self.source[ranged.range()]
```

Reply:
I went ahead and deleted the method...  :laughing: It's used in a single place where it's always a `TextRange`